JS-Algorithms/challenges.js

```diff
@@ -42,6 +54,37 @@ const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
   // YOUR CODE HERE...
+
+  if (index > 0) {
+    //Total
+    const totalElement = COUNTRY_NAMES.length;
+    let newArray = [];
+
+    //Find start index
+    let index2 = 0;
+    for (x = 0; x < index; x++) {
+      index2++;
+      if (index2 >= totalElement) {
+        index2 = 0;
+      }
+    }
+
```

Comment:
just a recommendation there are some array methods like slice, shift, join that could have helped with this exercise, please read more about array methods, I think this way you could reduce the quantity of for loops you used. You can find the list in the docs: https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Array or any other article related to array methods